proxy_auth_interceptor.go

```diff
@@ -139,14 +139,14 @@ func (interceptor *authInterceptor) login() Error {
 
 	claims := strings.Split(res.GetToken(), ".")
 	decClaims, gerr := base64.RawURLEncoding.DecodeString(claims[1])
```

Comment:
Let's remove the source of the issue here and replce all the `gerr` instances with `err`.

Reply:
@khaf, the assignment of the `Error` type to `err` and the `error` type to `gerr` may have required an additional error variable.

Reply:
I see; thanks for the update.